src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -247,20 +249,27 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     /*******************************************************************************************************
     * @description Returns valid contact primary affiliation fields from mappings
     * @param cf List of fields.
```

Comment:
We might want to update this parameter name too since you are changing it.

Reply:
Thanks for catching that. Updated it.

Reply:
Resolving.